### ISSUE TITLE
Config key "right_pane" has not been defined

### ISSUE THREAD

**Issue by vunhan** [jaap-karssenberg/zim-desktop-wiki#142]
I got this issue on ArchLinux, tried to removed old configs at `~/.config/zim` and reinstalled but this bug still happened.

```This is zim 0.66
Platform: posix
Locale: en_US UTF-8
FS encoding: UTF-8
Python: (2, 7, 13, 'final', 0)
Gtk: (2, 24, 31)
Pygtk: (2, 24, 0)
Zim revision is:
  branch: zim-trunk
  revision: 871 jaap.karssenberg@gmail.com-20170428103155-0xaqvtl2g53epq2y
  date: 2017-04-28 12:31:55 +0200

======= Traceback =======
  File "/usr/lib/python2.7/site-packages/zim/actions.py", line 240, in do_activate
    self.__get__(instance, instance.__class__)()
  File "/usr/lib/python2.7/site-packages/zim/actions.py", line 211, in func
    self.func(instance, active)
  File "/usr/lib/python2.7/site-packages/zim/gui/__init__.py", line 2079, in toggle_panes
    Window.save_uistate(self)
  File "/usr/lib/python2.7/site-packages/zim/gui/widgets.py", line 2727, in save_uistate
    self.uistate[key] = self.get_pane_state(key)
  File "/usr/lib/python2.7/site-packages/zim/config/dicts.py", line 541, in __setitem__
    raise KeyError('Config key "%s" has not been defined' % k)
KeyError: 'Config key "right_pane" has not been defined'

**Comment by jaap-karssenberg** [comment]
Already fixed in 09d843232f3f7d0050984a14cc300b089426943c

**Comment by vunhan** [comment]
ok, tks @jaap-karssenberg 👍
